src/quo2/components/drawers/documentation_drawers/view.cljs

```diff
@@ -14,18 +13,18 @@
    - `button-icon` button icon
    - `on-press-button` On press handler for the button
    - `shell?` use shell theme
-
    `content` Content of the drawer
    "
   [{:keys [title show-button? on-press-button button-label button-icon shell?]} content]
   [gesture/scroll-view
+   {:style                             style/outer-container
+    :always-bounce-vertical            false
+    :content-inset-adjustment-behavior :never}
    [rn/view {:style style/container}
     [text/text
```

Comment:
nit: while refactoring this. It seems that documentation drawer is using `title` component here. Can you take a look at updating to use it here as well rather than another implementation of it?